kyaml/setters/lookupkio.go

```diff
@@ -10,15 +10,18 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-var _ kio.Filter = &LookupSetters{}
+var _ kio.Filter = &LookupSettersDeprecated{}
 
-// LookupSetters identifies setters for a collection of Resources
-type LookupSetters struct {
+// LookupSettersDeprecated identifies setters for a collection of Resources
+type LookupSettersDeprecated struct {
```

Comment:
If we are deprecating this, we need to add some more explanation for why and what is the preferred solution.